CHANGELOG.md

```diff
@@ -3,6 +3,113 @@
+  and head.
+
+  This makes things consistent between the normal and repair paths, and improves concurrency while
+  repairing.
+
+- [#2253](https://github.com/tigerbeetle/tigerbeetle/pull/2253)
+
+  Reject prepares on the primary if its view isn't durable, much like solo clusters.
+
+  This solves a failing VOPR seed wherein a primary accepting prepares before making its log_view
+  durable exposes a break in its hash chain.
+
+- [#2259](https://github.com/tigerbeetle/tigerbeetle/pull/2259),
+  [#2246](https://github.com/tigerbeetle/tigerbeetle/pull/2246)
+
+  A few `sysctl`s and security frameworks (eg, seccomp) might block io_uring. Print out a more
+  helpful error message, rather than a generic "permission denied" or "system outdated".
+
+
+### Features
+
+- [#2234](https://github.com/tigerbeetle/tigerbeetle/pull/2234)
+
+  Introduce a new flag, `Account.flags.closed`, which causes an account to reject any further
+  transfers, except for voiding two-phase transfers that are still pending.
+
+  The account flag can be set during creation or through a closing transfer. In the latter case,
+  closed account can be re-opened by voiding or expiring the closing transfer.
+
+
+### Internals
+
+- [#2233](https://github.com/tigerbeetle/tigerbeetle/pull/2233)
+
+  Move multiversion build logic into `build.zig` from `release.zig`. This makes it much easier to
+  build multiversion binaries as part of a regular `zig build`, without having to invoke CI or
+  release process specific code that's normally part of `release.zig`.
+
+  It also makes it possible to build multiversion binaries on platforms that aren't x86_64 Linux.
+
+- [#2251](https://github.com/tigerbeetle/tigerbeetle/pull/2251)
+
+  Previously, TigerBeetle release numbers were based on a finicky conversion of GitHub's internal
+  action run number to a version number.
+
+  This was error prone, and difficult to reason about before hand (what would the given version
+  number for a release be?). Instead, make it so this very changelog is the source of truth for
+  the version number which is explicitly set.
+
+- [#2248](https://github.com/tigerbeetle/tigerbeetle/pull/2248)
+
+  Inside the VOPR, heal both WAL header sectors before replica startup.
+
+  Before, the logic would arbitrarily repair one WAL header sectors if both sectors are corrupt,
+  which could cause seeds to fail when only one one of the WAL header sectors is corrupt and the
+  other one is corrupted on startup.
+
+- [#2252](https://github.com/tigerbeetle/tigerbeetle/pull/2252)
+
+  Change `init` function signatures to allow for in-place initialization. This addresses the silent
+  stack growth caused by intermediate copy/move allocations during the initialization of large
+  objects.
+
+  Specifically, the `Forest` struct can grow indefinitely depending on the number of
+  `Grooves`/`IndexTrees` needed to support the StateMachine's custom logic, causing TigerBeetle to
+  crash during startup due to stack-overflow.
```

Comment:
In contrast, this one is well-worth including, as it illuminates a very interesting aspect of Zig, even though it's an internal change. 

I guess I mentally use the following criteria to decide between inclusion and truncation of a PR:

* Is there a category of readers which would be glad that the PR is included?